playlist2links

```diff
@@ -17,33 +22,44 @@ if [[ -n $(type -p wget) ]];
 then
 	COMMAND="wget -o /dev/null -O '${TMPFILE}.html' '$URL'"
 fi
-if [[ -z "$COMMAND" ]] && [[ -n $(type -p curl) ]];
+if [ -z "$COMMAND" ] && [[ -n $(type -p curl) ]];
 then
 	COMMAND="curl -s -o '${TMPFILE}.html' '$URL'"
 fi
-if [[ -z "$COMMAND" ]];
+if [ -z "$COMMAND" ];
```

Comment:
[[ is not defined by POSIX, and avoid word splitting and glob expansion are avoided by the quotes surrounding the variables, but yes, this might not be needed since the script is specifically for bash, and in any case bashism-related changes don't really belong to this commit...